lang/java/reef-bridge-java/src/main/java/org/apache/reef/javabridge/LibLoader.java

```diff
@@ -62,92 +62,58 @@ public void loadLib() throws IOException {
       this.loadBridgeDLL();
 
       // Load all DLLs in local
-      this.loadAllManagedDLLs(this.reefFileNames.getLocalFolder());
+      loadAllManagedDLLs(this.reefFileNames.getLocalFolder());
```

Comment:
I am :-) I removed the `this` for cases where a `static` method is called. And those methods were made static in the same change, as I saw that they really don't need object state.